config_complete.toml

```diff
@@ -219,6 +219,14 @@ Metrics = "honeycomb"
 # Not eligible for live reload.
 # RedisPassword = ""
 
+# RedisDatabase is an integer indicating the database number to use for the Redis instance storing
```

Comment:
Can we add somewhere in this comment that the value must be between 0-15.  I didn't actually know that redis worked like this haha

Reply:
Done.